jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -85,6 +85,14 @@ def step020_install_pe(SKIP_PE_INSTALL, script_dir, server_era) {
     }
 }
 
+def step025_collect_facter_data(job_name, script_dir) {
+    withEnv(["PUPPET_GATLING_SIMULATION_CONFIG=${PUPPET_GATLING_SIMULATION_CONFIG}",
```

Comment:
This will conflict with the approach in #76 which reworks how `PUPPET_GATLING_SIMULATION_CONFIG` is derived.  Not sure if you'd rather have this PR come in before that one - doesn't matter to me.